### PR TITLE
Fixed Project Files and Updated Readme

### PR DESCRIPTION
**Pull request by ryancj14**
1. Updated project .json files to show "yosys-vivado", to match the other updated specs in the python code. This toolchain is now run by specifying `--toolchain yosys-vivado`, so these .json files needed to be updated if it is to work.

2. Updated project names with underscores to instead use dashes. (This allows exhaust.py to run).

3. Updated Readme.md to reflect this and other recent updates to the code.

### CODE REVIEW

File: project/bram_n1.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "src/bram/rom.v",
         "src/bram/processing_unit.v",
         "src/bram/simpleuart.v",
-	"src/bram/scalable_proc.v"
+	    "src/bram/scalable_proc.v"
```
Comment (acomodi) [inline project/bram_n1.json:7]: indentation error

**Comment by kgugala** [comment]
there are some conflicts here. @ryancj14 can you rebase this PR and resolve them?

**Comment by mithro** [comment]
Merge if green!

**Comment by ryancj14** [comment]
It has an extra Travis CI Build here that is queued but won't start. Is there any way to get rid of that so we can merge?

**Comment by mithro** [comment]
I think we can just merge.

**Pull request merged**